### DISCUSSION TITLE
Trail of bits - recording

### DISCUSSION THREAD

**Discussion by planetf1** [open-quantum-safe/liboqs#1900] [General]
Was there any recording of the trail-of-bits call yesterday?
Unfortunately I wasn't able to make it due to a clash.

**Comment by dstebila** [comment]
> Was there any recording of the trail-of-bits call yesterday? Unfortunately I wasn't able to make it due to a clash.

I wasn't able to make a recording.  We did get a first draft of their report. They still have some editing to do before giving us a copy to share publicly, but I've emailed the TSC a copy and also a copy to you.
